x-pack/plugins/ingest_manager/common/services/decode_cloud_id.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// decodeCloudId decodes the c.id into c.esURL and c.kibURL
+export function decodeCloudId(
+  cid: string
+):
+  | {
+      host: string;
+      defaultPort: string;
+      elasticsearchUrl: string;
+      kibanaUrl: string;
+    }
+  | undefined {
+  // 1. Ignore anything before `:`.
+  const id = cid.split(':').pop();
+  if (!id) {
+    // throw new Error(`Unable to decode ${id}`);
+    // eslint-disable-next-line no-console
+    console.log(`Unable to decode ${id}`);
```

Comment:
Can we have a logger in our appContext? Looks like it's not to hard to do https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/plugin.ts/#L66

Reply:
I agree, but would like to skip for now to get this in. I can change to `console.debug` (like [here](https://github.com/elastic/kibana/blob/8a304623d4bf2e3be16f4c2cd25aa734b0308f5e/x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/create_datasource_page/index.tsx#L108)) or `console.error` if you'd prefer.

Reply:
`console.debug` is probably better  👍

Reply:
https://github.com/elastic/kibana/pull/65366/commits/1003d5992eca5ab90a04d8414175e24853f89969